### PR TITLE
add Rspec, Capybara and FactoryGirl

### PR DESCRIPTION
**Pull request by georgestam**
PR from issue #44. @vemv could you please have a quick look to see if I left out something?

I tested the factories in `rails c` and seem to be working fine. 

What are your reasons for using Codeship vs Travis? Cost only?

### CODE REVIEW

File: spec/factories/user_factory.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    default_location = 'barcelona'
+    address [default_location]
+    
+    transient do
+      linkedin_verif false
+      facebook_verif false
+    end
+    
+    after(:build) do |rec, eva|
+      rec.linkedin_verif = eva.linkedin_verif if eva.linkedin_verif
```
Comment (vemv) [inline spec/factories/user_factory.rb]: I think the transient/after blocks are entirely unnecessary in this case. transient is useful for attributes that the model does not have already

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    DatabaseCleaner.start
+    Warden.test_mode!
+  end
+  
+  config.before(:each, :js) do
+    page.driver.browser.manage.window.resize_to(1920, 1200)
+  end
+  
+  config.after(:each) do |example|
+    assert_js_ok if example.metadata[:js] && !example.metadata[:skip_js_check]
```
Comment (vemv) [inline spec/spec_helper.rb]: assert_js_ok will not work if you don't modify application.html.erb. Study that mechanism or discard this line

File: spec/factories/user_factory.rb (hunk 1 above)
Comment (georgestam) [inline spec/factories/user_factory.rb]: Agreed!

File: spec/spec_helper.rb (hunk 2 above)
Comment (georgestam) [inline spec/spec_helper.rb]: I have done a few changes in the application.html.erb and other places to make this work. Let me know what you think.

**Pull request merged**